<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LancelotPartConfigBase</class>
 <widget class="QWidget" name="LancelotPartConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>692</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="pageAdvanced">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="panelContents">
        <property name="title">
         <string comment="Contents of the applet">Activation:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioContentsActivationClick">
           <property name="text">
            <string>Click to activate items</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioContentsActivationExtender">
           <property name="text">
            <string>Use no-click interface</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="panelContentsExtenderPosition">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Extender Position</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radioContentsExtenderPositionLeft">
              <property name="text">
               <string comment="Position the extender left">Left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioContentsExtenderPositionRight">
              <property name="text">
               <string comment="Position the extender right">Right</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="panelIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Applet Icon:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="KIconButton" name="iconChooser">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20000</width>
             <height>64</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioIconActivationClick">
             <property name="text">
              <string>Show contents on &amp;click</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioIconActivationHover">
             <property name="text">
              <string>Show contents on icon &amp;hover (after a predefined interval of time)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="pageContents">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkShowSearchBox">
           <property name="text">
            <string>Show the search box</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listModels">
           <property name="styleSheet">
            <string>.QListWidget { border: none; background: transparent; }</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioContentsActivationExtender</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelContentsExtenderPosition</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioContentsActivationClick</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelContentsExtenderPosition</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
